{"name": "Ошибка при неверном id заказа", "status": "passed", "description": "Проверка ошибки 404 при использовании несуществующего id заказа", "steps": [{"name": "Отправка запроса на принятие заказа с некорректным id_order=123123123 и id_courier=569546", "status": "passed", "start": 1752420297931, "stop": 1752420298179}, {"name": "Проверка, что статус ответа равен 404", "status": "passed", "start": 1752420298179, "stop": 1752420298179}], "attachments": [{"name": "log", "source": "7032e346-d047-4da3-bfd1-8ff82c8cb0c6-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "cb238305-7811-4ff6-97e2-2c4f05da9c05-attachment.txt", "type": "text/plain"}], "start": 1752420297931, "stop": 1752420298179, "uuid": "5337c4ac-f2bb-4034-b36d-30e35cd8d7e2", "historyId": "18db3865cbbc6789b72efacbc51f5cdf", "testCaseId": "18db3865cbbc6789b72efacbc51f5cdf", "fullName": "Tests.test_accept_order.TestAcceptOrder#test_accept_order_with_wrong_id_order", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "DESKTOP-T2S04D8"}, {"name": "thread", "value": "8088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_accept_order"}]}